Add webpack shimming example to webpack docs (#590)
cloudflare/workers-docs: content/tooling/wrangler/webpack.md

}
```

## Shimming globals

Sometimes you want to bring your own implementation of an existing global API. You can do this by [shimming](https://webpack.js.org/guides/shimming/#shimming-globals) a third party module in its place as a webpack plugin.

> For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package and add a plugin entry to your webpack config.

Reviewer: does this need to be a global npm install or can it just be a local install for this worker? Assuming the latter, but it would be good to be explicit about that.

Author: i could also just say "add it to your package.json" and that would probably be clearer.

Reviewer: How about
```suggestion
For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package to install it locally and add it to your worker's package.json, then add a plugin entry to your webpack config.
```

> For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package and add a plugin entry to your webpack config.

Author: ```suggestion
For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - add it to your package.json and add a plugin entry to your webpack config.
```

> For example, to replace the runtime global `URL` class with the npm package `url-polyfill`, or your choice of third party package, `npm i` the package, and then add a plugin entry to your webpack config

Reviewer 2: ```suggestion
For example, to replace the runtime global `URL` class with the npm package `url-polyfill` - or your choice of third party package - `npm i` the package and add a plugin entry to your webpack config.
```
